features/environment.py

```diff
@@ -653,13 +652,9 @@ def build_debs_from_sbuild(context: Context, series: str) -> List[str]:
         )
 
         with emit_spinner_on_travis("Building debs from local source... "):
-            output_deb_dir = os.path.join(tempfile.gettempdir(), series)
             deb_paths = build_debs(
-                series=series,
-                output_deb_dir=output_deb_dir,
-                cache_source=context.config.cache_source,
+                series=series, cache_source=context.config.cache_source
             )
-            context.config.debs_path = output_deb_dir
 
         logging.info("--- sbuild has finished building the packages")
```

Comment:
I think we should move this alert inside the `build_debs`  function, otherwise even if we reuse the packages we will still emit it

Reply:
Good point! I will just move this and then merge

Reply:
Actually `build_debs` already logs "Succesfully ran sbuild" so I will just remove this line